app/controllers/post_controller.rb

```diff
@@ -0,0 +1,9 @@
+class PostsController < ApplicationController
+  def index
+    @posts = Post.all
+  end
+
+  def show
+    @post = Post.find_by_id(params[:id])
+  end
+end
```

Comment:
- [ ] Same thing for the `posts_controller`, it would be nice to add a corresponding view for each of the `posts_controller` actions to follow the project requirement.
![Screenshot from 2023-12-19 19-19-33](https://github.com/BHObserver/blog-app/assets/91142432/9fc7651b-b940-499f-895e-f9c7749a1e3d)
**Example:**
```rb
    # app/views/posts/index.html.erb
    # app/views/posts/show.html.erb
```